docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
 
-- Make sure you're logged into slow.pics
-- Open the developer menu, go to Application at the top, then Storage on the left, open Local Storage and select slow.pics. Copy the browserId.
-- Still in the developer menu, go to Network, refresh the page to populate the list, select the slow.pics entry and scroll until you see SLPSESSION and copy the ID.
-- Back in VS-Preview, go to settings, Comp, and paste the two IDs in the relevant boxes.
+==- 2 - Framing parameters
 
-+++ Semi-auto
+#### Random
 
-### Marking the frames
+The *Random* parameter sets the number of frames to skip before screenshotting. *This is a setting used when creating [automatic](#automatic-badge-variant-secondary-text-fastest-option)/[semi-automatic](#semi-automatic-badge-icon-variant-primary-text-recommended) comparisons.*
 
-- Press `Shift` & `Right Arrow` keys to move forward a set amount of frames (configurable in playback section of the toolbar)
-- `Number` keys to switch between video sources and compare quality.
-- Press `Control` & `Space` keys to mark the frame numbers. (Configurable in cfg files)
-- Aim to screenshot a variety of scenes, light/dark, low/high motion, etc.
-- Try and match frame type when screenshotting, e.g. all sources on a `B` frame, single frame jump comes in handy for when they don't match (`Arrow keys`).
-- If a source file does not have `B` frames for you to match, you should skip matching frame type entirely for that source. This is usually true for Crunchyroll WEB-DLs, which have no `B` frames.
+For example, a value of `48` means VSPreview will take a screenshot for every 48 frames in all sources.
 
-### Uploading the comp
+==- 3 - Picture type
 
-- Once you are done marking frames, click the `comp` button in the bottom right of the toolbar.
-- Tick the `public` box to make the comparison available on the slow.pics homepage.
-- Name the collection appropriately for Slow.pics, get the TMDB ID from <https://www.themoviedb.org/> and paste it in the TMDB box, with it correctly set to Movie/TV depending on the content.
-- Set the tag to the same name as listed on TMDB.
-- Set "Delete After" to your desired value, by default comps expire after 2 years of not being viewed.
-- Under `random` you may specify an additional frame amount for VS-Preview to pick at random, recommended to help fill the comparison with more frames.
-- Untick `Current frame` as you likely don't want that included.
-- Make sure your PNG compression is set to 0 (Max) under Settings -> Main
+Sets the frame type to capture. *This is a setting used when creating [automatic](#automatic-badge-variant-secondary-text-fastest-option)/[semi-automatic](#semi-automatic-badge-icon-variant-primary-text-recommended) comparisons.*
 
-**If you wish to upload to a slow.pics account for comp history and the ability to edit comps do the following:**
+Only checked frames are captured. For instance, if `I` is unchecked, then VSPreview will only capture `P` or `B` frames.
```

Comment:
~~recommend using P/B frames since they are more useful than I frames for comps. although sometimes it's impossible to select and pick frame types when all the sources don't have matching ones~~ you covered this ahead